app/models/document-type.js

```diff
@@ -1,5 +1,5 @@
 import { hasMany } from '@ember-data/model';
 import Concept from './concept';
 export default class DocumentType extends Concept {
-  @hasMany('document', { async: true }) documents;
+  @hasMany('document', { inverse: null, async: true }) documents;
```

Comment:
Was getting a deprecation warning (for ember v5), you have to mention the inverse of a relation.
It was one of the only relation that did not have this.
ex: 
in `document-container` model:
`@hasMany('piece', { inverse: 'documentContainer', async: true }) pieces;`
in `piece` model:
`@belongsTo('document-container', { inverse: 'pieces', async: true }) documentContainer;`
The explicit mentioning helps the ember store update the list of pieces on documentContainer of a new piece sets the belongsTo to that document-container.

For concepts however, we often don't specify the inverse.
It should not have any negative effects.

Also, from looking at this code. We wouldn't be able to get `documentType.documents` in ember because it does not match the definition in domain models.

`@hasMany('document-container', { inverse: null, async: true }) documentContainers;`
and maybe `inverse: 'concept'` but normally we don't do that (since concept does not really know us in `concept.js` model